precovery/precovery_db.py

```diff
@@ -207,14 +210,18 @@ def _check_frames(
             n = 0
             for obs in self.frames.iterate_observations(f):
                 n += 1
-                if obs.matches(exact_ephem, tolerance):
+                matches, dra, ddec, distance = obs.matches(exact_ephem, tolerance)
```

Comment:
Would it be cleaner if we redefine `obs.matches` to `obs.distance` and have this function return `dra, ddec, distance`? We can then delegate the check against `tolerance` in this for-loop rather than returning the boolean.

Reply:
Done, I've changed the matches function to distance and move the check against tolerance to the for-loop in [f4ad4d5](https://github.com/B612-Asteroid-Institute/precovery/pull/8/commits/f4ad4d53b5363eefa9b51ba681a4795edb5f481b).

Reply:
Done, I've changed the matches function to distance and moved the check against tolerance to the for-loop in [f4ad4d5](https://github.com/B612-Asteroid-Institute/precovery/pull/8/commits/f4ad4d53b5363eefa9b51ba681a4795edb5f481b).